docker-compose.yml

```diff
@@ -50,6 +50,9 @@ services:
       EUREKA_CLIENT_SERVICEURL_DEFAULTZONE: "http://discovery-server:8761/eureka/"
       SPRING_CLOUD_CONFIG_LABEL: "main"
       SPRING_CLOUD_CONFIG_URI: "http://config-server:8888"
+      SPRING_CLOUD_REPO: "https://raw.githubusercontent.com/NDPH-ARTS/mts-trial-deployment-config/"
```

Comment:
why do you need these 2 lines?
the cloud config server gets the profile and knows which repo and search path to use.

I dont think the role service need this info